examples/task-manager/seed-data/data.ts

```diff
@@ -44,28 +44,24 @@ export const tasks = [
   {
     label: 'Query content over GraphQL🔎',
     isComplete: false,
-    finishBy: '2021-03-21T05:41:37.000Z',
```

Comment:
Why removing these?

Reply:
Mostly because they weren't being used - when they are used, they should be future dates, so this existing string doesn't make sense.